style.css

```diff
@@ -0,0 +1,485 @@
+* {
+  font-family: "Poppins", sans-serif;
+  padding: 0;
+  margin: 0;
+}
+
+.logo {
+  color: #344563;
+}
+
+body {
+  width: fit-content;
+  background-color: #e5e5e5;
+  padding: 0;
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  overflow-x: hidden;
+}
+
+header {
+  background-color: white;
+  width: 100%;
+}
+
+.backgroundimg {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  background-image: url("/image/background.png");
+  background-repeat: no-repeat;
+  background-size: contain;
+  background-color: white;
+  border-radius: 0 0 0 120px;
+  height: 680px;
+  width: 100%;
+}
+
+#cards {
+  display: grid;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+}
+
+.card {
+  display: grid;
+  flex-direction: column;
+  margin: 60px auto;
+  background-color: white;
+  border-radius: 16px;
+  width: 86%;
+  height: fit-content;
+  padding: 24px 30px;
+}
```

Comment:
- [ ] As I have earlier suggested, I think it would be better to use flexbox to align each card as specified in the project requirement. I can see you are still trying to use `grid`. It's very important to gain mastery of both `flexbox` and `grid` which is the reason why you are to use both of them 😄